Doc/reference/datamodel.rst

```diff
@@ -1733,10 +1741,10 @@ Basic customization
 
 .. method:: object.__str__(self)
```

Comment:
I thought we had a doc bug years ago where we came up with a convention to differentiate `object.__str__` (actual method that exists in the `object` class) and `someclass.__str__` (method you can define in your class, part of the datamodel)

I guess we didn’t…